styles/modrinth/catppuccin.user.css

```diff
@@ -165,6 +176,17 @@
       border-color: @text transparent transparent;
       color: @text;
     }
+    // header button hover
+    .btn-wrapper:hover {
+      a,
+      button {
+        --tw-brightness: brightness(1) !important;
+      }
+    }
+    // loading indicator
```

Comment:
```suggestion
```
Generally, no need for comments like these when the name of the class/id/attr/etc is self explanatory (in this case the class contains both words of the comment).